core/ibft.go

```diff
@@ -455,12 +457,42 @@ func (i *IBFT) handleRoundChangeMessage(view *proto.View) *proto.RoundChangeCert
 		isValidFn,
 	)
 
-	if !i.backend.HasQuorum(view.Height, msgs, proto.MessageType_ROUND_CHANGE) {
+	// group messages by round and take the messages for the highest round
```

Comment:
Is it possible to have messages from different rounds here? `func (ms *Messages) GetValidMessages`  calls `getProtoMessages` which is defined like:
```
func (ms *Messages) getProtoMessages(
	view *proto.View,
	messageType proto.MessageType,
) protoMessages {
	heightMsgMap := ms.getMessageMap(messageType)

	// Check if the round map is present
	roundMsgMap, found := heightMsgMap[view.Height]
	if !found {
		return nil
	}

	return roundMsgMap[view.Round]
}
```

Reply:
You're right
Fixed a0be0455d61addf55465a6b28934c417acfdf1d0